app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -600,6 +601,22 @@ export default Component.extend(ComposerUpload, {
     });
   },
 
+  _warnHereMention(hereCount) {
+    if (!hereCount || hereCount === 0) {
+      return;
+    }
+
+    schedule("afterRender", () => {
```

Comment:
I dont think schedule + later is needed here, you probably just need later, and ensure this is not destroyed/destroying

Reply:
I found this strange too, but the same pattern is used for all the other notices. I was under the impression that it was done like this with a purpose.

Do you want me to change all of them?

Reply:
I guess we can start by only using later here, and change others in another PR if that doesn’t cause issues